ros_buildfarm/config/doc_build_file.py

```diff
@@ -167,14 +167,21 @@ def __init__(self, name, data):  # noqa: D107
         # user host and docroot have default of uploading to the repo machine
         # next to the debs
         self.upload_user = data.get('upload_user', 'jenkins-agent')
-        self.upload_host = data.get('upload_host', 'repo')
+        self.upload_host = data.get('upload_host', None)
```

Comment:
Note that I'm changing from a default of `repo` (which was bogus anyway), to a default of `None`.

Reply:
`repo` is a valid SSH host for the Jenkins publish-over-ssh configuration and corresponds to the host running the building_repository agent. Since this agent hosts an http server it's used as the default docs host for custom build farms.

I expect that this default is consistent across most of the doc jobs and would suggest either not changing it or changing it everywhere.

Reply:
Oh, OK.  Sorry, that is not clear at all here.

Well, the main reason I changed it is so that I can do error checking down below to make sure only one of `upload_host` and `upload_repository_url` was specified per-job.  I guess I can change the logic here so that if *neither* is specified, it falls back to an `upload_host` of "repo".  Does that make sense?

Reply:
I went ahead and did this in 4c374cf12ffbf265b248771fc9d9fc0c2dc38e0e